tests/unit/rangeslider/rangeslider_core.js

```diff
@@ -139,30 +139,33 @@
 		rangeslider.rangeslider( { disabled: true } );
 
 		ok( !!inputFirst.attr( "disabled" ),  "first input is disabled" );
-		ok( inputFirst.hasClass( "mobile-slider-disabled"),
+		assert.hasClasses( inputFirst, "mobile-slider-disabled",
 			"first input gets mobile-slider-disabled" );
 		ok( !!sliderFirst.attr( "aria-disabled" ), "first slider is aria-disabled" );
-		ok( sliderFirst.hasClass( "ui-state-disabled"), "first slider has ui-state-disabled" );
+		assert.hasClasses( sliderFirst, "ui-state-disabled", 
+			"first slider has ui-state-disabled" );
 		ok( !!inputLast.attr( "disabled" ),  "last input is disabled" );
-		ok( inputLast.hasClass( "mobile-slider-disabled"),
+		assert.hasClasses( inputLast, "mobile-slider-disabled", 
 			"last input gets mobile-slider-disabled" );
 		ok( !!sliderLast.attr( "aria-disabled" ), "last slider is aria-disabled" );
-		ok( sliderLast.hasClass( "ui-state-disabled"), "last slider has ui-state-disabled" );
+		assert.hasClasses( sliderLast, "ui-state-disabled", "last slider has ui-state-disabled" );
 
 		rangeslider.rangeslider( {disabled: false } );
 
 		deepEqual( !!inputFirst.attr( "disabled" ), false, "first input is enabled" );
```

Comment:
Do not use deepEqual here its only for arrays, objects, regular expressions, dates and functions http://api.qunitjs.com/deepEqual/